api/README.md

```diff
@@ -0,0 +1,166 @@
+# Clspv - C API wrapper
```

Comment:
This belongs in docs as `C_API.md` or a similar name.

Reply:
Ack.